src/Core/src/Handlers/TimePicker/TimePickerHandler.Android.cs

```diff
@@ -43,12 +44,23 @@ void onTimeSetCallback(object? obj, TimePickerDialog.TimeSetEventArgs args)
 
 				if (_dialog != null)
 				{
+					_dialog.DismissEvent -= dismiss;
 					_dialog = null;
 				}
 			}
 
 			var dialog = new TimePickerDialog(Context!, onTimeSetCallback, hour, minute, Use24HourView);
 
+			dismiss = (sender, e) =>
+			{
+				if (VirtualView != null)
+				{
+					VirtualView.IsFocused = false;
+				}
+				dialog.DismissEvent -= dismiss;
+			};
+			dialog.DismissEvent += dismiss;
```

Comment:
If the dialog here never fired the `DismissEvent`, does something else need to unsubscribe? Maybe `DisconnectHandler()`?

Reply:
It makes sense! I added a commit where I unsubscribe from such events in DisconnectHandler for both date and time pickers